core/cap-0045.md

```diff
@@ -391,8 +391,41 @@ price <img style="transform: translateY(0.1em); background: white;" src="https:/
 Specifically, <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=%5Cfrac%7Bp_%7Bfull%7D%7D%7Bp_%7Blim%7D%7D%20%3D%20%5Cfrac%7B1%7D%7B(1-2%5E%7B-%5Csigma%7D)%7D">.
 
 It doesn't matter whether you linearly interpolate the asset sold or the asset bought. The two quantities are linearly related according to the budget
-constraint `pX + Y = pX' + Y'`, so linear interpolation of one implies linear
-interpolation of the other.
+constraint <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=p_x%20x%20%2B%20p_y%20y%20%3D%20p_x%20x'%20%2B%20p_y%20y'">, so linear interpolation of one asset implies linear interpolation of the other.
+
+To derive the demand results in the previous section, consider an offer selling asset `X` (we also use `X` to denote the amount of `X` being offered) for asset `Y` at limit price <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=p_%7Blim%7D%3DP">. At any point, the budget must be conserved (subject to a fee `F`):
+
+<div align="center"><img style="background: white;" src="https://render.githubusercontent.com/render/math?math=y%5Ccdot%20p_%7By%7D(1-F)%20%3D%20p_%7Bx%7D%5Ccdot%20x"></div>
+
+When <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=p_x%20%5Cle%20p_%7Blim%7D">, no trade will happen, demand for `Y` is 0, and demand for `X` is the full amount of X valued at <img style="transform: translateY(0.1em); background: white;" src="https://render.githubusercontent.com/render/math?math=p_%7Bx%7DX">.
```

Comment:
This is wrong, you're using both `p(X)` and `p(Y)` here so you always need to take ratios. Same applies in all of these statements, and the graph.

Reply:
I updated. Thanks for point it out.